tensornetwork/matrixproductstates/mps.py

```diff
@@ -451,7 +531,7 @@ def apply_one_site_gate(self, gate: Tensor, site: int) -> None:
         gate_node, self.nodes[site],
         name=self.nodes[site].name).reorder_edges(edge_order)
 
-  def measure_local_operator(self, ops: List[Tensor],
+  def measure_local_operator(self, ops: List[Union[BaseNode, Tensor]],
```

Comment:
The `measure_` methods can go in the `BaseMPS` class, right?

Reply:
yes that's right. Moved them up a class